lib/elixir/test/elixir/macro_test.exs

```diff
@@ -313,6 +313,10 @@ defmodule MacroTest do
     """
   end
 
+  test "not in to string" do
+    assert Macro.to_string(quote do: false not in []) == "false not in []"
```

Comment:
I'd put parentheses around `false not in []` to be consistent with other examples.